trie/statistics/trieStatistics.go

```diff
@@ -143,8 +146,18 @@ type TrieStatsDTO struct {
 
 // ToString returns the collected statistics as a string array
 func (tsd *TrieStatsDTO) ToString() []string {
+	addressConverter, err := pubkeyConverter.NewBech32PubkeyConverter(addressLength, log)
```

Comment:
we should not create a bech32 pubkey converter each time we call this String method. Suggestion to pass to this String method a converter interface